src/renderer/components/+workloads-deployments/deployments.store.ts

```diff
@@ -18,7 +18,7 @@ export class DeploymentStore extends KubeObjectStore<Deployment> {
 
   loadMetrics(deployment: Deployment) {
     const pods = this.getChildPods(deployment);
-    return podsApi.getMetrics(pods, deployment.getNs(), "").then(metrics =>
+    return podsApi.getMetrics(pods, deployment.getNs()).then(metrics =>
```

Comment:
This is the line that is the fix. The rest is just a little bit of cleanup

Reply:
Please split cleanup to another PR. Impossible to say (by just looking at the diff) if cleanup causes unwanted side-effects.

Reply:
@Nokel81 Please add a bit more description or test case how it will solve a problem? I wonder if it's somehow related to my guess here https://github.com/lensapp/lens/issues/807#issuecomment-699911167 ?

Reply:
Will do

Reply:
@aleksfront This reverts the `podsApi.getMetrics(...)` back to its default `selector` (3rd argument) which limits the metrics gotten to `"pod, namespace"`. It seems that even if we do explicitly ask for those things, there is some duplication if we don't select just `"pod, namespace"`.

Now, tbh, I don't have a lot of experience with prometheus and I came to try this solution because I was looking at the generated queries for metrics and this was the sole location that wasn't filtering/selecting on anything.